docs/sources/setup-grafana/configure-security/configure-authentication/saml/index.md

```diff
@@ -168,38 +172,54 @@ Grafana supports user authentication through Okta, which is useful when you want
+| `allowed_organizations`                                    | No       | List of comma- or space-separated organizations. User should be a member of at least one organization to log in.                                                                                             |                                                       |
+| `org_mapping`                                              | No       | List of comma- or space-separated Organization:OrgId:Role mappings. Organization can be `*` meaning "All users". Role is optional and can have the following values: `Viewer`, `Editor` or `Admin`.          |                                                       |
+| `role_values_editor`                                       | No       | List of comma- or space-separated roles which will be mapped into the Editor role                                                                                                                            |                                                       |
+| `role_values_admin`                                        | No       | List of comma- or space-separated roles which will be mapped into the Admin role                                                                                                                             |                                                       |
+| `role_values_grafana_admin`                                | No       | List of comma- or space-separated roles which will be mapped into the Grafana Admin (Super Admin) role                                                                                                       |                                                       |
+| `name_id_format`                                           | No       | Name ID Format to request within the SAML assertion                                                                                                                                                          | `urn:oasis:names:tc:SAML:2.0:nameid-format:transient` |
 
 ### Signature algorithm
 
 > **Note:** Available in Grafana version 7.3 and later.
 
 The SAML standard recommends using a digital signature for some types of messages, like authentication or logout requests. If the `signature_algorithm` option is configured, Grafana will put a digital signature into SAML requests. Supported signature types are `rsa-sha1`, `rsa-sha256`, `rsa-sha512`. This option should match your IdP configuration, otherwise, signature validation will fail. Grafana uses key and certificate configured with `private_key` and `certificate` options for signing SAML requests.
 
+### Specify user's Name ID
+
+> **Note:** Available in Grafana version 9.1 and later.
```

Comment:
because we version our docs, we don't need to include this note. If this is available in other versions, should we backport the docs? We're currently backporting as far back as v9.3

Reply:
We can remove it, was following the rest of the page's examples.